### PR TITLE
add pulpcore mash to script

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: koji/collection-mash-split.py [hunk 1]
```diff
@@ -470,6 +470,14 @@ def main():
                 el8_candlepin.name = 'katello-candlepin-{}-el8'.format(version)
                 mashes.append(el8_candlepin)
 
+    elif collection == 'pulpcore':
+        CONFIG["gitloc"] = "https://github.com/theforeman/pulpcore-packaging.git"
+        git_tag = "rpm/{}".format(version)
```
Comment (ekohl) [inline koji/collection-mash-split.py:475]: This suggests there may be deb packaging in the future. Or is this only to mirror foreman-packaging?

Comment (evgeni) [reply koji/collection-mash-split.py:475]: To mirror f-packaging. The alternative would be to call the branches `PULPCORE-X.Y` or something. I don't mind either way.

Comment (evgeni) [reply koji/collection-mash-split.py:475]: Mirroring has the benefit that we know the Jenkins jobs can parse that ;)

Comment (ekohl) [reply koji/collection-mash-split.py:475]: I'm ok with that. Do you intend to have a master branch similar to foreman-packaging which just serves a README about the layout (and is the default branch) or will you keep updating the default branch over time to the latest version?

Comment (evgeni) [reply koji/collection-mash-split.py:475]: Good question I didn't think about yet.

Keeping the latest branch as default sounds good.

**Review by ekohl — APPROVED** [review]
Please capture that policy somewhere. Either in the pulpcore-packaging's README.md or in some procedure in tool_belt come to mind.

**Comment by evgeni** [comment]
> Please capture that policy somewhere. Either in the pulpcore-packaging's README.md or in some procedure in tool_belt come to mind.

ack, noted down a todo item to do so.

**Pull request merged**